marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/mapping.spec.tsx

```diff
@@ -119,6 +127,17 @@ describe("Mapping", () => {
     advancedSettingsDialog.saveSettings(mapStep).click();
     advancedSettingsDialog.saveSettings(mapStep).should("not.be.visible");
 
+    //verify mapping step with duplicate name cannot be created
+    cy.waitUntil(() => curatePage.addNewStep().click());
+    createEditMappingDialog.setMappingName(mapStep);
+    createEditMappingDialog.setSourceRadio("Query");
+    createEditMappingDialog.setQueryInput("test");
+    createEditMappingDialog.saveButton().click({force: true});
+    //error message should be displayed instead of step details auto open
+    cy.findByLabelText(`${mapStep}-details-header`).should("not.be.visible");
+    loadPage.duplicateStepErrorMessage();
+    loadPage.confirmationOptions("OK").click();
```

Comment:
Can we add similar validations for Match and Merge Steps. And once we click "OK" can we also check whether the modal is not visible anymore.

Reply:
Thanks @ngodugu-marklogic , I've added the same checks for Match and Merge and also added a check that the modal is no longer visible afterwards